geoips/filenames/base_paths.py

```diff
@@ -180,6 +200,28 @@ def initialize_paths():
             "TC_DECKS_DB": "longterm_files/tc/tc_decks.db",
             "TC_DECKS_DIR": "longterm_files/tc/decks",
         },
+        paths["GEOIPS_DATA_CACHE_DIR"]: {
+            "GEOIPS_DATA_CACHE_DIR_LONGTERM_GEOLOCATION_DYNAMIC": (
+                "longterm/geolocation/dynamic"
+            ),
+            "GEOIPS_DATA_CACHE_DIR_LONGTERM_GEOLOCATION_STATIC": (
+                "longterm/geolocation/static"
+            ),
+            "GEOIPS_DATA_CACHE_DIR_SHORTTERM_GEOLOCATION_SOLAR_ANGLES": (
+                "shortterm/geolocation/solar"
+            ),
+            "GEOIPS_DATA_CACHE_DIR_SHORTTERM_CALIBRATED_DATA": (
+                "shortterm/calibrated_data"
+            ),
+        },
+        paths["SATPY_DATA_CACHE_DIR"]: {
+            "SATPY_DATA_CACHE_DIR_SHORTTERM_CALIBRATED_DATA": (
+                "shortterm/calibrated_data"
+            ),
+            "SATPY_DATA_CACHE_DIR_SHORTTERM_GEOLOCATION_SOLAR_ANGLES": (
+                "shortterm/geolocation/solar"
+            ),
+        },
```

Comment:
proxyvis caching should be updated to use these base paths, rather than the hard coded locations that are there now.